MAUI/DataGrid/paging.md

```diff
@@ -297,9 +295,249 @@ public partial class MainPage : ContentPage
+                   AutoEllipsisMode="After"
+                   Source="{Binding OrdersInfo}">
+</pager:SfDataPager>
+{% endhighlight %}
+{% highlight c# %}
+public partial class MainPage : ContentPage
+{
+    public MainPage()
+    {
+        InitializeComponent();
+        dataPager.AutoEllipsisMode = DataPagerEllipsisMode.After;
+    }
+}
+{% endhighlight %}
+{% endtabs %}
+
+![AutoEllipsisMode .NET MAUI DataGrid](Images\paging\maui-datagrid-autoelipsismode.png)
+
+## AutoEllipsisText
+
+The data pager allows the user to customize the text to be displayed in the auto ellipsis mode by using `SfDataPager.AutoEllipsisText` property. The default value of `AutoEllipsisText` is set to `…`.
+
+{% tabs %}
+{% highlight xaml %}
+<pager:SfDataPager x:Name="dataPager"
+                   PageSize="15"
+                   AutoEllipsisMode="After"
+                   AutoEllipsisText="***"
+                   Source="{Binding OrdersInfo}">
+</pager:SfDataPager>
+{% endhighlight %}
+{% highlight c# %}
+public partial class MainPage : ContentPage
+{
+    public MainPage()
+    {
+        InitializeComponent();
+        dataPager.AutoEllipsisMode = DataPagerEllipsisMode.After;
+        dataPager.AutoEllipsisText = "***";
+    }
+}
+{% endhighlight %}
+{% endtabs %}
+
+## Move to pages programmatically
+
+### MoveToFirstPage
+
+The data pager allows the users to programmatically navigate to the first page using the `MoveToFirstPage()` method.
+
+### MoveToLastPage
+
+The data pager allows the users to programmatically navigate to the last page using the `MoveToLastPage()` method.
+
+### MoveToNextPage
+
+The data pager allows the users to programmatically navigate to the next page using the `MoveToNextPage()` method.
+
+### MoveToPreviousPage
+
+The data pager allows the users to programmatically navigate to the previous page using the `MoveToPreviousPage()` method.
+
+### MoveToPage(Int32)
+
+The data pager allows the users to programmatically navigate to the desired page using the `MoveToPage(Int32)` method.
+
+### MoveToPage(Int32, Int32, Boolean)
+
+The data pager allows the users to programmatically navigate to the desired page from the current page using the `MoveToPage(Int32, Int32, Boolean)` method.
+
+## Orientation
+
+By default, `SfDataPager` scrolls horizontally. However, `SfDataPager.Orientation` property allows users to customize the pager to scroll vertically or horizontally according to their preferences.
+
+{% tabs %}
+{% highlight xaml %}
+<pager:SfDataPager x:Name="dataPager"
+                   PageSize="15"
+                   Orientation="Vertical"
+                   Source="{Binding OrdersInfo}">
+</pager:SfDataPager>
+{% endhighlight %}
+{% highlight c# %}
+public partial class MainPage : ContentPage
+{
+    public MainPage()
+    {
+        InitializeComponent();
+        dataPager.Orientation = DataPagerScrollOrientation.Vertical;
+    }
+}
+{% endhighlight %}
+{% endtabs %}
+
+## Events
+
+### PageChanged

```

Comment:
PageChanging event should be written first.

Reply:
Addressed the concerns provided.